src/components/redux/books/books.jsx

```diff
@@ -0,0 +1,39 @@
+// Reducer
```

Comment:
Kindly ensure to define action types, ensure the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK`